### PR TITLE
feat: change default yaml flow style to False

### PR DESCRIPTION
**Pull request by RadxaYuntian**
The default flow style is hard to read for non-trivial system configurations.

[Here](https://github.com/yaml/pyyaml/issues/199#issuecomment-420425200) is how different flow style looks like.

This is a real output we are currently generating with bdebstrap, which is hard to read: [config.yaml.txt](https://github.com/bdrung/bdebstrap/files/14174160/config.yaml.txt)

Readability is important because we have customers who want to customize our systems. In the past we have to teach them how to use our build system, but we plan to just let them make changes to the generated `config.yaml` and build it.

Extension name changed to pass GitHub upload rule.

### CODE REVIEW

**Comment by bdrung** [comment]
The change looks good. Can you add a test case for this change?

**Comment by RadxaYuntian** [comment]
I'm currently in vacation. I'll revisit this PR after 18th.

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /*.egg-info/
 /.pybuild/
 __pycache__/
+.coverage
```
Comment (bdrung) [inline .gitignore:6]: Can you put that into a separate commit?

**Comment by bdrung** [comment]
Thanks.

**Review by bdrung — APPROVED** [review]
